tutorials/live_templates/template_support.md

```diff
@@ -3,70 +3,128 @@ title: 1. Adding Live Template Support
 ---
 <!-- Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
```

Comment:
there's a lot of repeating "IntelliJ Platform" on this page, maybe we could strip a few places

Reply:
Yes, 12 times is a lot. I cut that down to 7.